Readme.md

```diff
@@ -108,37 +108,64 @@ See the Concurrent Updates section for details how to implement this in a custom
 ## Type Marshaling
 
 `ddbus` includes fancy templated methods for marshaling D types in and out of DBus messages.
-All DBus-compatible basic types work (except dbus path objects and file descriptors).
-Any forward range can be marshalled in as DBus array of that type but arrays must be taken out as dynamic arrays.
-Structures are mapped to `Tuple` from `std.typecons`.
+All DBus-compatible basic types work (except file descriptors).
+Any forward range can be marshaled in as DBus array of that type but arrays must be taken out as dynamic arrays.
+
+As per version 2.3.0, D `struct` types are fully supported by `ddbus`. By default all public fields of a structure are marshaled. This behavior can be changed by UDAs. Mapping DBus structures to a matching instance of `std.typecons.Tuple`, like earlier versions of `ddbus` did, is also still supported.
```

Comment:
can you explain how it is still supported?

Reply:
Left this as is, but added some more explanation to the type mapping table below.